.github/workflows/check_repos.yml

```diff
@@ -0,0 +1,85 @@
+# Test official instructions for installing Dangerzone
+# ====================================================
+#
+# The installation instructions have been copied from our INSTALL.md file.
+# NOTE: When you change either place, please make sure to keep the two files in
+# sync.
```

Comment:
Maybe add a note here that because commands run as root, use of `sudo` is unnecessary?

Reply:
Fixed in 70e7caaebdf69e66c6242797b9570a38b02c79f1.